fakebook/src/app/component/like-view/like-view.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { LikeService } from 'src/app/service/like.service';
+
+@Component({
+  selector: 'app-like-view',
+  templateUrl: './like-view.component.html',
+  styleUrls: ['./like-view.component.css']
+})
+export class LikeViewComponent implements OnInit {
+  @Input() count!: number;
+  @Input() postId!: number;
+  @Input() liked!: boolean;
```

Comment:
the fact that we're just getting initial values from the parent component and not really expecting them to change afterwards, instead modifying the values on our own initiative, should be documented

Reply:
I don't even know for this one